#Service to Ping the robot

---
int64 ret
string msg